vividus/src/main/java/org/vividus/spring/StringToFormatConverter.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.spring;
+
+import java.lang.reflect.Field;
+
+import org.apache.commons.lang3.Validate;
+import org.jbehave.core.reporters.Format;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.util.ReflectionUtils;
+
+public class StringToFormatConverter implements Converter<String, Format>
```

Comment:
is it possible to use `PropertyEditor` instead? (you can implement `PropertyEditorSupport`)

Reply:
It was one of the first tries to fix the issue with the list of formats, but unfortunately it did not work

Reply:
Issue was discussed privately